apps/emqx_machine/src/emqx_machine_boot.erl

```diff
@@ -128,41 +129,62 @@ reboot_apps() ->
     BaseRebootApps ++ ConfigApps.
 
 basic_reboot_apps() ->
-    ?BASIC_REBOOT_APPS ++
-        [
-            emqx_prometheus,
-            emqx_modules,
-            emqx_dashboard,
-            emqx_connector,
-            emqx_gateway,
-            emqx_resource,
-            emqx_rule_engine,
-            emqx_bridge,
-            emqx_management,
-            emqx_retainer,
-            emqx_exhook,
-            emqx_authn,
-            emqx_authz,
-            emqx_slow_subs,
-            emqx_auto_subscribe,
-            emqx_plugins,
-            emqx_psk
-        ] ++ basic_reboot_apps_edition(emqx_release:edition()).
-
-basic_reboot_apps_edition(ce) ->
-    [emqx_telemetry];
-basic_reboot_apps_edition(ee) ->
-    [
-        emqx_license,
-        emqx_s3,
-        emqx_ft,
-        emqx_eviction_agent,
-        emqx_node_rebalance,
-        emqx_schema_registry
-    ];
-%% unexcepted edition, should not happen
-basic_reboot_apps_edition(_) ->
-    [].
+    PrivDir = code:priv_dir(emqx_machine),
+    RebootListPath = filename:join([PrivDir, "reboot_lists.eterm"]),
```

Comment:
https://www.erlang.org/doc/man/code.html#priv_dir-1

Reply:
:thinking:

Reply:
Sorry, that was a brain fart.